test/test_unit_api.py

```diff
@@ -42,9 +47,19 @@
     "version": 4
 }
 
-
-MOCK_HOSTFACTORY_OBJECT = CreateTokenData(host_factory="some_host_factory", cidr="1.2.3.4,0.0.0.0", days=1)
+MOCK_HOSTFACTORY_OBJECT = CreateTokenData(host_factory="some_host_factory", cidr="1.2.3.4,0.0.0.0",
+                                          days=1)
 MOCK_HOSTFACTORY_WITHOUT_CIDR_OBJECT = CreateTokenData(host_factory="some_host_factory", days=1)
+MockCredentials = CredentialsData(login='myuser', password='mypass')
+
+
+def create_api(url='http://localhost', account='default',
+               ssl_verification_mode=SslVerificationMode.SELF_SIGN,
+               cert_path=DEFAULT_CERTIFICATE_FILE):
+    return Api(conjurrc_data=ConjurrcData(conjur_url=url, account=account, cert_file=cert_path),
+               ssl_verification_mode=ssl_verification_mode,
+               credentials_provider=FileCredentialsProvider())
```

Comment:
If newline is needed between params, either in signature or method call, separate *each* param in newline.
Also, I prefer not calling methods within methods.
```suggestion
def create_api(url='http://localhost',
               account='default',
               ssl_verification_mode=SslVerificationMode.SELF_SIGN,
               cert_path=DEFAULT_CERTIFICATE_FILE):
    conjurrc_data = ConjurrcData(conjur_url=url, account=account, cert_file=cert_path)
    return Api(conjurrc_data=conjurrc_data,
               ssl_verification_mode=ssl_verification_mode,
               credentials_provider=FileCredentialsProvider())
```